lib/metadata/MetadataDiscovery.ts

```diff
@@ -453,7 +453,7 @@ export class MetadataDiscovery {
   }
 
   private defineBaseEntityProperties(meta: EntityMetadata): void {
-    const base = this.metadata.get(meta.extends);
+    const base = meta.extends && this.metadata.get(meta.extends);
 
     if (!meta.extends || !base) {
```

Comment:
we can now simplify this 

```suggestion
    if (!base) {
```